test/tests/LayerStatesTest.coffee

```diff
@@ -158,24 +185,24 @@ describe "LayerStates", ->
 		# it "should be a no-op to change to the current state", ->
 		# 	layer = new Layer
 		# 	layer.states.stateA = {x: 100}
-		# 	layer.switchInstant "stateA"
+		# 	layer.stateSwitch "stateA"
 		# 	animation = layer.animate "stateA", time: 0.05
 		# 	assert.equal(animation, null)
```

Comment:
Because the behaviour changed. We actually do always return an animation now so that the events get called and you can skip a null check.